app/models/audio_file_template.rb

```diff
@@ -20,4 +23,32 @@ def set_defaults
     self.length_minimum ||= 0
     self.length_maximum ||= 0
   end
+
+  def validate_audio_file(file)
+    [audio_length_errors(file), label_mismatch_errors(file)].join(' ').strip
+  end
+
+  private
+
+  def audio_length_errors(file)
+    length_errors = ''
+    shorter_than_min = file.length < length_minimum
+    longer_than_max = length_maximum != 0 && file.length > length_maximum
+
+    if shorter_than_min || longer_than_max
+      length_errors << "Audio file #{file.position} '#{file.label}' is " +
+                       "#{file.length.to_time_string}, but the '#{file.label}' " +
+                       "must be between #{length_minimum.to_time_string} " +
+                       "and #{length_maximum.to_time_string}."
+    end
+    length_errors
+  end
+
+  def label_mismatch_errors(file)
```

Comment:
In what scenario would this error be important?

If you re-labeled one of your audio-file-templates, this would cause Feeder to unpublish every episode of your podcast.  I was thinking we would only unpublish them if you re-durationed one of the templates, and the file didn't meet the new requirement.